### ISSUE TITLE
Jenkins plugin Json or P12 key are not found and saved empty

### ISSUE THREAD

**Issue by anniekvandijk** [jenkinsci/google-oauth-plugin#13]
When I try to push a Docker image to GGloud i get this stacktrace

```
com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials$AccountIdNotSetException
	at com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials.getGoogleCredential(GoogleRobotPrivateKeyCredentials.java:181)
	at com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials.getGoogleCredential(GoogleRobotPrivateKeyCredentials.java:45)
	at com.google.jenkins.plugins.credentials.oauth.GoogleRobotCredentials.getAccessToken(GoogleRobotCredentials.java:101)
	at com.google.jenkins.plugins.googlecontainerregistryauth.GoogleContainerRegistryCredentialModule.getToken(GoogleContainerRegistryCredentialModule.java:73)
```
When I open my credentials the file is gone. 

![image](https://user-images.githubusercontent.com/18325609/33323343-828af9c6-d44c-11e7-8d19-e344df98bd5a.png)

Key files are generated but not found. Also the files are empty

![image](https://user-images.githubusercontent.com/18325609/33323533-0b319dca-d44d-11e7-8840-e3c1f14bf3a0.png)

Jenkins for Windows ver. 2.73.3
Google OAuth Credentials plugin 0.5

**Comment by DzianisH** [comment]
I have totally the same issue. I've tried to populate *.json and *.p12 files on Jenkins manually but it wasn't help me.

**Comment by proskehy** [comment]
I have the same issue, is anyone looking into this?

**Comment by sortidocop** [comment]
I have the same issue to, nobody find a bypass ? or a way to put manualy the json file ? ty.

**Comment by mneil** [comment]
Version 0.5 doesn't seem to have this issue. Check that the user you are running Jenkins on has permission to change file permissions.

**Comment by proskehy** [comment]
How do I go about checking that? Jenkins is a Windows service and I start the job as an admin user.

**Comment by barrymac** [comment]
@mneil I tried this with 0.6-SNAPSHOT I built and manually installed and still got the problem, any patch needed? 

@proskehy I ran `mvn install` in this repository and copied the hpi file to the plugins directory in place of the the installed one.

**Comment by soartseng** [comment]
Today, I got the 0.6 version updated announcement.
In 0.6 version, I have the same problem in Jenkins "Free-style" project.
Bit I can success to checkout in Jenkins "pipeline" project.
I just copy the "checkout syntax" in others project and replace 2 parts of the syntax to the right:
1. credentialsid -> replace to your right credentials id that you created Google Service Account from private key in jenkins.
2. url ->replace to your google source repository url

**Comment by stephenashank** [comment]
Resolved with #19 and #48.
